apps/finance/app/src/App.js

```diff
@@ -125,137 +110,61 @@ class App extends React.Component {
     const { newTransferOpened } = this.state
 
     return (
-      <Root.Provider>
-        <PublicUrl.Provider url="./aragon-ui/">
-          <BaseStyles />
-          <Main>
-            <AppView
-              padding={contentPadding}
-              appBar={
-                <AppBar>
-                  <BreakPoint to="medium">
-                    <AppBarContainer>
-                      <Title>
-                        <MenuButton onClick={this.handleMenuPanelOpen} />
-                        <TitleLabel>Finance</TitleLabel>
-                      </Title>
-                      <ButtonIcon
-                        onClick={this.handleNewTransferOpen}
-                        title="New Transfer"
-                        css={`
-                          width: auto;
-                          height: 100%;
-                          padding: 0 20px 0 10px;
-                          margin-left: 8px;
-                        `}
-                      >
-                        <img src={newTransferIcon} alt="" />
-                      </ButtonIcon>
-                    </AppBarContainer>
-                  </BreakPoint>
-                  <BreakPoint from="medium">
-                    <AppBarContainer style={{ padding: '0 30px' }}>
-                      <Title>
-                        <TitleLabel>Finance</TitleLabel>
-                      </Title>
-                      <Button
-                        mode="strong"
-                        onClick={this.handleNewTransferOpen}
-                      >
-                        New Transfer
-                      </Button>
-                    </AppBarContainer>
-                  </BreakPoint>
-                </AppBar>
-              }
-            >
-              {balances.length > 0 && (
-                <SpacedBlock>
-                  <Balances balances={balances} />
-                </SpacedBlock>
-              )}
-              {transactions.length > 0 && (
-                <SpacedBlock>
-                  <Transfers transactions={transactions} tokens={tokens} />
-                </SpacedBlock>
-              )}
-              {balances.length === 0 && transactions.length === 0 && (
-                <EmptyScreen>
-                  <EmptyStateCard
-                    icon={<img src={addFundsIcon} alt="" />}
-                    title="Add funds to your organization"
-                    text="There are no funds yet - add funds easily"
-                    actionText="Add funds"
-                    onActivate={this.handleNewTransferOpen}
-                  />
-                </EmptyScreen>
-              )}
-            </AppView>
-            <SidePanel
+      <div css="min-width: 320px">
```

Comment:
Should this be part of `AppLayout`?

Reply:
I wanted to, but `SidePanel` need the `min-width` too 😔. A solution could be to have a `sidePanels` prop on `AppLayout`, but it would make the panels appear before the content, so I left it like this.